openam-core-rest/src/main/java/org/forgerock/openam/core/rest/CoreRestRouteProvider.java

```diff
@@ -104,7 +115,9 @@ public void addResourceRoutes(ResourceRouter rootRouter, ResourceRouter realmRou
                 .forVersion(2, 1)
                 .toCollection(Key.get(IdentityResourceV2.class, Names.named("AgentsResource")))
                 .forVersion(3, 0)
-                .toCollection(Key.get(IdentityResourceV3.class, Names.named("AgentsResource")));
+                .toCollection(Key.get(IdentityResourceV3.class, Names.named("AgentsResource")))
+                .forVersion(4, 0)
+                .toCollection(Key.get(IdentityResourceV4.class, Names.named("AgentsResource")));
```

Comment:
This PR is not about agents. So is there a reason for introducing this new version?